arangod/Aql/ExecutionBlock.cpp

```diff
@@ -32,14 +32,16 @@
 
 using namespace arangodb::aql;
   
-ExecutionBlock::ExecutionBlock(ExecutionEngine* engine, ExecutionNode const* ep)
-    : _engine(engine),
-      _trx(engine->getQuery()->trx()),
-      _exeNode(ep),
-      _pos(0),
-      _done(false),
-      _profile(engine->getQuery()->queryOptions().profile),
-      _getSomeBegin(0) {
+ExecutionBlock::ExecutionBlock(
+    ExecutionEngine* engine,
+    ExecutionNode const* ep) noexcept
```

Comment:
shouldn't be made noexcept, because an ExecutionBlock has a member `_buffer`, which is an `std::queue`. `std::deque`s constructor is not `noexcept`, so it may throw, e.g. because it failed to allocate some initial memory.

Reply:
yes, I've overlooked this